src/agent/src/message_queue_utils.hpp

```diff
@@ -15,3 +17,9 @@ boost::asio::awaitable<std::string> GetMessagesFromQueue(std::shared_ptr<IMultiT
 void PopMessagesFromQueue(std::shared_ptr<IMultiTypeQueue> multiTypeQueue, MessageType messageType);
 
 void PushCommandsToQueue(std::shared_ptr<IMultiTypeQueue> multiTypeQueue, const std::string& commands);
+
+std::optional<command_store::Command> GetCommandFromQueue(std::shared_ptr<IMultiTypeQueue> multiTypeQueue);
+
+bool PopCommandFromQueue(std::shared_ptr<IMultiTypeQueue> multiTypeQueue);
```

Comment:
Since we don't need to check the return value, I think this function can return void.

Reply:
Done